raspberry_pi_experiments/power_consumption_benchmark.py

```diff
@@ -0,0 +1,85 @@
+import service
```

Comment:
Imports should be grouped in the following order:

standard library imports
related third party imports
local application/library specific imports

From https://www.python.org/dev/peps/pep-0008/#imports